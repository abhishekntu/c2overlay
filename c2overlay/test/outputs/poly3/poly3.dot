digraph graphname {
N12 [color="black", label="mul_Imm_80_N8_N12"];
N13 [color="black", label="N9N7_mul_Imm_2_N9_add_N7_N13"];
N10 [color="black", label="N11N17_mul_Imm_25_N11_sub_N17_N10"];
N11 [color="black", label="sqr_N16_N11"];
N14 [color="black", label="O0_N18_N14"];
N8 [color="black", label="N13N14_mul_Imm_15_N13_add_N14_N8"];
N9 [color="black", label="N10N15_mul_Imm_11_N10_sub_N15_N9"];
N1 [color="black", label="I0_N1_N1"];
N2 [color="black", label="I1_N2_N2"];
N3 [color="black", label="I2_N3_N3"];
N4 [color="black", label="I3_N4_N4"];
N5 [color="black", label="I4_N5_N5"];
N6 [color="black", label="I5_N6_N6"];
N7 [color="black", label="mul_N12_N7"];
N12 -> N10;
N13 -> N9;
N10 -> N8;
N11 -> N9;
N8 -> N14;
N9 -> N8;
N1 -> N13;
N2 -> N7;
N3 -> N11;
N4 -> N7;
N5 -> N13;
N6 -> N12;
N7 -> N10;
}
