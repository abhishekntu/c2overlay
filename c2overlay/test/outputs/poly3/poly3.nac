procedure foo(in s32 t,in s32 u,in s32 v,in s32 x,in s32 y,in s32 z,out s32 D_2451)
{
  localvar s32 D_2450;
  localvar s32 t1_1;
  localvar s32 t1_2;
  localvar s32 t1_3;
  localvar s32 t1_4;
  localvar s32 t1_5;
  localvar s32 t1_6;
  localvar s32 t1_7;
  localvar s32 t1_8;
  localvar s32 t1_9;
  localvar s32 t1_10;
  localvar s32 t2_1;
  localvar s32 t2_2;
  localvar s32 t2_3;
  localvar s32 t2_4;
  localvar s32 t2_5;
  localvar s32 t2_6;
  localvar s32 t2_7;
  localvar s32 t2_8;
  localvar s32 t3_1;
  localvar s32 t3_2;
  localvar s32 t3_3;
  localvar s32 t3_4;
  localvar s32 t3_5;
  localvar s32 t3_6;
  localvar s32 t3_7;
  localvar s32 t4_1;
  localvar s32 t4_2;
  localvar s32 t4_3;
  localvar s32 t4_4;
  localvar s32 t5_1;
  localvar s32 t5_2;
  localvar s32 t5_3;
  localvar s32 t6_1;
  localvar s32 t6_2;
  localvar s32 t6_3;
  localvar s32 t7_1;
  localvar s32 t7_2;
  localvar s32 t8_1;
  localvar s32 t8_2;
  localvar s32 t9_1;
  localvar s32 t10_1;
  localvar s32 t11_1;
L0005:
  t1_1 <= sqr v;
  t1_2 <= mul y,u;
  t2_1 <= mul t1_1,11;
  t2_2 <= mul t1_2,25;
  D_2450 <= mul t,2;
  t3_1 <= add D_2450,x;
  t3_2 <= mul z,80;
  t4_1 <= sub t3_1,t2_1;
  t4_2 <= sub t2_2,t3_2;
  t5_1 <= mul t4_1,15;
  D_2451 <= add t5_1,t4_2;
}
