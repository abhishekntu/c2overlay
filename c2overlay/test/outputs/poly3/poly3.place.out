Netlist file: test.net   Architecture file: test.arch.xml
Array size: 3 x 3 logic blocks

#block name	x	y	subblk	block number
#----------	--	--	------	------------
N1		1	4	0	#0
N2		4	1	0	#1
N3		0	1	0	#2
N4		3	0	0	#3
N5		0	3	0	#4
N6		4	2	0	#5
out:N14		2	4	0	#6
N12_blk		3	2	0	#7
N13_blk		1	3	0	#8
N10_blk		2	1	0	#9
N11_blk		1	1	0	#10
N8_blk		2	2	0	#11
N9_blk		1	2	0	#12
N7_blk		3	1	0	#13
